CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ We try only to add languages once they have some usage on GitHub. In most cases
 To add support for a new language:
 
 0. Add an entry for your language to [`languages.yml`][languages]. Omit the `language_id` field for now.
-0. Add a grammar for your language: `script/add-grammar https://github.com/JaneSmith/MyGrammar`. Please only add grammars that have [one of these licenses][licenses].
+0. Add a grammar for your language: `script/add-grammar https://github.com/JaneSmith/MyGrammar` and download its license: `script/licensed`. Please only add grammars that have [one of these licenses][licenses].
```

Comment:
The call to `script/licensed` is included in `script/add-grammar` so it shouldn't be needed. Since you also had an issue with the ordering in `grammars.yml`, I'm guessing you attempted to add the grammar manually...?